### ISSUE TITLE
`quarto inspect` of a file which includes a project level format extension fails

### ISSUE THREAD

**Issue by dragonstyle** [quarto-dev/quarto-cli#6353]
1) Clone repo <https://github.com/cwickham/manuscript-template-rstudio>
2) At terminal, in repo root, run `quarto inspect notebooks/explore-earthquakes.qmd`
3) Error:

```
ERROR: Unable to read the extension 'agu'.
Please ensure that you provided the correct id and that the extension is installed.

Stack trace:
Please ensure that you provided the correct id and that the extension is installed.
    at loadExtension (file:///Applications/quarto/bin/quarto.js:41583:15)
    at Object.extension (file:///Applications/quarto/bin/quarto.js:41482:34)
    at readExtensionFormat (file:///Applications/quarto/bin/quarto.js:75101:50)
    at resolveFormats (file:///Applications/quarto/bin/quarto.js:75047:41)
    at eventLoopTick (ext:core/01_core.js:181:11)
    at async renderContexts (file:///Applications/quarto/bin/quarto.js:74421:21)
    at async renderFormats (file:///Applications/quarto/bin/quarto.js:74472:26)
    at async inspectConfig (file:///Applications/quarto/bin/quarto.js:96121:29)
    at async Command.fn (file:///Applications/quarto/bin/quarto.js:96169:20)
    at async Command.execute (file:///Applications/quarto/bin/quarto.js:8100:13)
```

This is occurring because `quarto inspect <filepath>` isn't providing the project context when inspecting the file. In this case, however, the extension was installed at the project level, so the file can't be properly inspected without providing this context.

**Comment by cderv** [comment]
Fixed by https://github.com/quarto-dev/quarto-cli/commit/1e3334114e0cb91592740dd819400a9e7548e542 not sure why it wasn't closed 🤔

**Comment by mianzg** [comment]
Hi, I am running into this error again with version 1.4.383 on macOS.

**Comment by cderv** [comment]
Let's reopen then. Thanks for reporting back. 

@dragonstyle I see the commit referenced above is not present in `main` yet. Sorry for missing that. 
Is that in a PR somewhere ?

**Comment by mcanouil** [comment]
@cderv https://github.com/quarto-dev/quarto-cli/pull/6354/commits The commit to look for in `main` is [dbf0df6](https://github.com/quarto-dev/quarto-cli/commit/dbf0df64d67c8f11eb5aaad295950a69cc24dbfe)

**Comment by cderv** [comment]
Thanks a lot @mcanouil ! Got bitten again by a squashed PR ... 🤦  

So a fix should be available in `main` and shipped with `[v1.4.273](https://github.com/quarto-dev/quarto-cli/releases/tag/v1.4.273)`

I tested the example at https://github.com/quarto-dev/quarto-cli/issues/6353#issue-1825044603 and it is working fine. 

@mianzg I close this PR as there is a fix merged for it already. Please do open a new issue with your latest problem. Thank you !
